feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/domain/interfaces/CoinPriceRepository.kt

```diff
@@ -7,6 +7,10 @@ import retrofit2.HttpException
 import kotlin.jvm.Throws
 import kotlin.time.Duration
 
+enum class Range {
+    DAY, WEEK, MONTH, YEAR, MAX
+}
+
 interface CoinPriceRepository {
```

Comment:
Just noticed that the repository is located in domain layer. I think we should move it to data